operator/pkg/controllers/etcd/pod.go

```diff
@@ -28,6 +28,11 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	instanceTypeLabelKey          = "node.kubernetes.io/instance-type"
+	instanceTypeLabelDefaultValue = "m5.16xlarge"
```

Comment:
`16xlarge` to be the default might be too big instance for many scenarios, but we can revisit this. Not a blocker at the moment